### PR TITLE
Added info about Admin UI region availability

### PR DESCRIPTION
**Pull request by KristinaPeterson**
*Issue #, if available:*

*Description of changes:*
Admin UI isn't available in every region where the Amplify Console is available. Added this information and a link to the currently supported regions. Reviewed by @swaminator before submitting.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by thaddmt — APPROVED** [review]

File: docs/start/getting-started/fragments/next/hosting.md [hunk 1]
```diff
@@ -1,49 +1,28 @@
-You've successfully built an Amplify app with Next.js! Now that you've built something, it's time to deploy it to the web!
+You've successfully built an Amplify app with Next.js! Now that you've built something, it's time to deploy it to the web with Amplify Console!
 
 > **Note**: Amplify Console is adding support for Server-Side Rendering (SSR).
 > [Vote & comment on this issue](https://github.com/aws-amplify/amplify-console/issues/412) to show your support!
 
-## Using [Serverless Next.js Component](https://github.com/serverless-nextjs/serverless-next.js)
+## Add hosting to your app
+Next.js SSR apps can only be set up with automatic continuous deployments. In this guide we'll cover how to deploy and host your SSR web apps to quickly share with others. If you want to learn more about continuous deployment, please see [this guide](https://docs.aws.amazon.com/amplify/latest/userguide/multi-environments.html#standard).
```
Comment (swaminator) [inline docs/start/getting-started/fragments/next/hosting.md:7]: You can deploy Next.js SSR apps with AWS Amplify's CI/CD and hosting service.

File: docs/start/getting-started/fragments/next/hosting.md [hunk 2]
```diff
@@ -1,49 +1,28 @@
...
-
 ```console
-  nextamplified:
-    appUrl:         https://••••••••••.cloudfront.net
-    bucketName:     XXXXXX-XXXXXXX
-    distributionId: XXXXXXXXXXXXXX
+? Select the plugin module to execute: # Hosting with Amplify Console (Managed hosting with custom domains, Continuous deployment)
+? Choose a type: # Continuous deployment (Git-based deployments)
 ```
+The Amplify Console will open to your deployed backend environment. Finish setting up continuous deployment by connecting your app's frontend to the Git repository where your project is stored.
```
Comment (swaminator) [inline docs/start/getting-started/fragments/next/hosting.md:20]: We need to be a bit more descriptive here. Can we add an image?

**Review by swaminator — CHANGES_REQUESTED** [review]

File: docs/start/getting-started/fragments/next/hosting.md (hunk 1 above)
Comment (KristinaPeterson) [reply docs/start/getting-started/fragments/next/hosting.md]: I added this language in my updated commit.

File: docs/start/getting-started/fragments/next/hosting.md (hunk 2 above)
Comment (KristinaPeterson) [reply docs/start/getting-started/fragments/next/hosting.md]: I added additional screenshots for the work flow in the Amplify console.

**Review by swaminator — APPROVED** [review]

**Pull request merged**